tests/tt_metal/tt_metal/unit_tests_common/watcher/test_noc_sanitize.cpp

```diff
@@ -89,7 +88,8 @@ void RunTestOnCore(WatcherFixture* fixture, Device* device, CoreCoord &core, boo
             output_dram_noc_xy.y = 16;
             break;
         case SanitizeAlignment:
-            l1_buffer_addr += 1;
+            l1_buffer_addr += 16; // This is illegal because reading DRAM->L1 needs DRAM alignment
```

Comment:
is there a test which covers the new odd case that should pass, eg:  read from DRAM at offset 16 bytes and write to L1 at offset 16 bytes ?

Reply:
The implementation here doesn't have support for this case currently. Two ways to add: have an exception for this case (a little hacky but not so bad to implement), or fully switch to relative alignment checking (involves going through all `noc_sanitize_*` calls and fixing any that aren't paired, nontrivial).

Is there demand for this case specifically vs general relative alignment? If so I'll add an exception for it, if not we can leave it for when we switch to relative alignment checking

Reply:
ok, makes sense, leave it as is for now as we currently don't take advantage of the relative alignment